src/series/Castlevania.yml

```diff
@@ -86,27 +86,53 @@ randomizers:
     updated-date: 1900-01-01
     added-date: 1900-01-01
 -   games:
-    - 'Castlevania: Symphony of the Night (PlayStation / Saturn)'
-    identifier: 3snowp7im's Area Randomizer
+    - 'Castlevania: Symphony of the Night (PlayStation)'
+    identifier: Castlevania: SOTN Randomizer (Launcher Application by SacredLucy)
+    url: https://github.com/LuciaRolon/SotNRandomizerLauncher/releases
+    comment: 'The premiere, community-driven SOTN Randomizer. Includes Relic, Item, Color, Enemy Stats, and Shop Price randomizers. Also Includes a Boss Rush Mode. Browser version here: <https://sotn.io/> GitHub here: <https://github.com/3snowp7im/SotN-Randomizer>'
+    updated-date: 2024-12-03
+    added-date: 2024-12-03
+    opensource: true
+-   games:
```

Comment:
you appear to have added this entry twice?

Reply:
We have three different methods of access for this and a tutorial site. This randomizer is probably one of the most robust and diverse in delivery method.

Reply:
oh, wait, you added it three times? ideally someone else can chime in but I don't see why multiple entries are needed for a single randomizer. especially since you currently link all three methods of access in all three entries. there's no need to take up all this space on the list